protocol/v2/ssv/runner/validator_registration.go

```diff
@@ -9,10 +9,20 @@ import (
 	"github.com/bloxapp/ssv-spec/qbft"
 	specssv "github.com/bloxapp/ssv-spec/ssv"
 	spectypes "github.com/bloxapp/ssv-spec/types"
-	"github.com/bloxapp/ssv/protocol/v2/types"
 	ssz "github.com/ferranbt/fastssz"
 	"github.com/pkg/errors"
 	"go.uber.org/zap"
+
+	"github.com/bloxapp/ssv/logging/fields"
+	"github.com/bloxapp/ssv/protocol/v2/qbft/controller"
+	"github.com/bloxapp/ssv/protocol/v2/types"
+)
+
+// TODO: This is a reasonable default, but we should probably make this configurable.
+//
+//	Discussion here: https://github.com/ethereum/builder-specs/issues/17
+const (
+	gasLimit = 30_000_000
```

Comment:
this shows up in a few different places around the code, same as the batch size. even if we're not adding it as a configuration right now, I'd put the consts in one file and read it from there to prevent confusion when one is changed.